cpmonitor/models.py

```diff
@@ -602,6 +595,54 @@ def __str__(self) -> str:
         return self.alt_description + " - Quelle: " + self.source
 
 
+class LocalGroup(models.Model):
+    class Meta:
+        verbose_name = "Lokalgruppe"
+        verbose_name_plural = "Lokalgruppen"
+
+    city = models.OneToOneField(
```

Comment:
In theory, there could also be one local group caring about several cities. @kleins05 Only thought about that after the merge. What do you think? Worth an issue? Should this be done soon or could such a change be done easily later?

Reply:
You are right in theory, but I think it is not worth an issue until we actually have a need. Won't be more work then than it is now.